infrastructure-playbooks/purge-cluster.yml

```diff
@@ -222,9 +222,7 @@
       timeout: 500
 
   - name: remove data
-    file:
-     path: /var/lib/ceph
-     state: absent
+    command: rm -rf /var/lib/ceph/*
```

Comment:
what's wrong with the `file` module?

Reply:
if /var/lib/ceph is mounted from a device, it can not be removed. whereas, ansible file module try to remove this folder rather than remove it, then error happends.

Reply:
Makes sense.